referenceframe/frame.go

```diff
@@ -20,6 +20,9 @@ import (
 	"go.viam.com/rdk/utils"
 )
 
+// comparisonDelta is the amount two floats can differ before they are considered different.
+const comparisonDelta = 1e-5
```

Comment:
Minor nit: I see this changed from `epsilon` to `comparisonDelta`, personally I think the former is more technically accurate.